CONTRIBUTING.md

```diff
@@ -170,25 +210,33 @@ mod tests {
 }
 ```
 
-Tests should be concise and non-repetitive: a function doesn't need to be tested with different
-inputs multiple times if variance in the input doesn't affect the functionality. The logic of a code
-path only needs to be uniquely tested once; testing the same conditions multiple times has no
-benefit.
+Tests should be concise and non-repetitive: a function doesn't need to be tested
+with different inputs multiple times if variance in the input doesn't affect the
+functionality. The logic of a code path only needs to be uniquely tested once;
+testing the same conditions multiple times has no benefit.
+
+We strive to follow these guidelines, however, some parts of the code have not
+been updated to do so.
```

Comment:
Perhaps we can have a general part of this guide that talks about how any portions of the codebase that have not yet been updated can be updated.